tests/Directory.Build.props

```diff
@@ -7,7 +7,7 @@
     <SignAssembly>True</SignAssembly>
     <IsPackable>false</IsPackable>
     <AssemblyOriginatorKeyFile>../../build/MSAL.snk</AssemblyOriginatorKeyFile>
-    <LangVersion>10.0</LangVersion>
```

Comment:
Shouldn't be necessary. We can remove the line has tests/Directory.Build.props include the parent one?

Reply:
Correct. Having it in the main Directory.Build.props should be enough

Reply:
Removed all applicable ones thanks Jenny and Jean-Marc!!